src/components/errors/ErrorNetworkMismatch.tsx

```diff
@@ -4,12 +4,21 @@ import { getChainName } from '@dcl/schemas/dist/dapps/chain-id'
 import { ErrorContainer, ErrorDetails, ErrorImage } from './Error'
 import errorImage from '../../images/errors/robotsmiling.png'
 import { switchToChainId } from '../../eth/provider'
+import { track } from '../../utils/tracking'
 import './errors.css'
 
 export interface ErrorNetworkMismatchProps {
   extra: Record<string, any>
```

Comment:
The same problem pointed out [in line 14](https://github.com/decentraland/explorer-website/pull/258/commits/61913d62b0486a7064c9a25c1a7fe2442ca5cb8b#diff-1ac65470fc2c1a0d5c5cab3c4d9ee33c59a1c27e700b17bedbd0f59443c96634R14) is present here by the way. This seems unnecessarily loose as a type